tiff.yaml

```diff
@@ -0,0 +1,53 @@
+# Generated from https://git.alpinelinux.org/aports/plain/main/tiff/APKBUILD
+package:
+    name: tiff
+    version: 4.5.0
+    epoch: 0
+    description: Provides support for the Tag Image File Format or TIFF
+    target-architecture:
+        - all
+    copyright:
+        - paths:
+            - '*'
+          attestation: TODO
+          license: libtiff
+environment:
+    contents:
+        packages:
+            - busybox
+            - ca-certificates-bundle
+            - build-base
+            - automake
+            - autoconf
+            - zlib-dev
+            - libjpeg-dev
+            - libwebp-dev
+            - zstd-dev
+            - python3
+            - cmake
+            - samurai
+pipeline:
+    - uses: fetch
+      with:
+        expected-sha256: dafac979c5e7b6c650025569c5a4e720995ba5f17bc17e6276d1f12427be267c
+        uri: https://download.osgeo.org/libtiff/tiff-${{package.version}}.tar.xz
+    - uses: autoconf/configure
+    - uses: autoconf/make
+    - uses: autoconf/make-install
+    - uses: strip
+subpackages:
+    - name: tiff-doc
+      pipeline:
+        - uses: split/manpages
+      description: tiff manpages
+    - name: tiff-dev
+      pipeline:
+        - uses: split/dev
+      dependencies:
+        runtime:
+            - tiff
+            - zlib-dev
+            - libjpeg-turbo-dev
```

Comment:
Ah - this ought to change to `libjpeg-dev`

Reply:
Fixed!